test/unit/steps/compliance/have-a-mrn/HaveAMRN.test.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const { expect } = require('test/util/chai');
+const { Reference } = require('@hmcts/one-per-page/forms');
+const HaveAMRN = require('steps/compliance/have-a-mrn/HaveAMRN');
+const DateUtils = require('utils/DateUtils');
+const paths = require('paths');
+const answer = require('utils/answer');
+
+describe('HaveAMRN.js', () => {
+
+    let haveAMRN;
+
+    beforeEach(() => {
+
+        haveAMRN = new HaveAMRN({
+            journey: {
+                steps: {
+                    DWPIssuingOffice: paths.compliance.dwpIssuingOffice,
+                    HaveContactedDWP: paths.compliance.haveContactedDWP
+                }
+            }
+        });
+    });
+
+    describe('get path()', () => {
+
+        it('returns path /have-a-mrn', () => {
+            expect(haveAMRN.path).to.equal(paths.compliance.haveAMRN);
```

Comment:
When accessing class **static** functions it's conventional and more readable to do the following: `HaveAMRN.path ` e.g. `ClassName.functionName`.